### PR TITLE
Request Open-Canalyzer OCA-V1 USB PID 4242

### PR DESCRIPTION
**Pull request by xor-gate**
Hi all,

Thanks for this awesome project. I would like to request the PID 4242 for our Open Source CAN-bus analyzer.

### CODE REVIEW

File: 1209/4242/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: pid
+title: OCA-V1
+owner: Open-Canalyzer
+license: MIT
+site: http://open-canalyzer.github.io
```
Comment (peternewman) [inline 1209/4242/index.md]: This doesn't work, do you mean http://open-canalyzer.github.io/docs/ ?

Comment (xor-gate) [inline 1209/4242/index.md]: You are right, I must have mistyped wil fix ASAP

**Comment by xor-gate** [comment]
Fixed

**Comment by peternewman** [comment]
LGTM

**Comment by xor-gate** [comment]
Are we good to go?

**Comment by peternewman** [comment]
Probably, I'm only interested in Markdown. @Arachnid will need to approve the PR.

**Comment by Arachnid** [comment]
Thanks for your Pull Request!

In order to issue you a PID, both the hardware and software components of your project need to be licensed under a recognised open-source license, with source code released.

Unfortunately, I don't see any source files for the hardware - only gerbers - and can't find any source code at all. Can you include these in your repository, please?

**Comment by xor-gate** [comment]
ping @maartentamboer

**Comment by maartentamboer** [comment]
Hi,

Our firmware is here: https://github.com/Open-Canalyzer/oca-fw-chibios
PC user space driver: https://github.com/Open-Canalyzer/liboca
For the hardware we use circuitmaker which is a cloud based program. After registration one can open/fork the hardware design: https://circuitmaker.com/Projects/CD5B5E2D-5168-4AD4-B4CE-60B1731C09AE

**Comment by xor-gate** [comment]
@maartentamboer maybe it should be better noted where the hardware design sources are and which license we use (OSHW). At https://github.com/Open-Canalyzer/docs/tree/master/hw

**Comment by Arachnid** [comment]
Thanks ,but you need to update the pull request to point to these repositories - people will never be able to find them otherwise.

**Comment by Arachnid** [comment]
Please open a new PR when you're ready.

**Pull request closed without merging**